src/main/java/brandkon/brand/Brand.java

```diff
@@ -0,0 +1,59 @@
+package brandkon.brand;
+
+import brandkon.category.Category;
+import jakarta.persistence.*;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+public class Brand {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    private String imageUrl;
+    private String guidelines;
+
+    public Brand() {
```

Comment:
사소한 거긴 한데, 개발자가 직접 사용할 생성자는 아니라서 public으로 두기보다 접근 가능 단계를 한 단계 낮추어 protected로 두면 개발을 좀 더 잘해 보이는 효과가..
```suggestion
    protected Brand() {
```